docker/Dockerfile.cuda110.x86_64.deps

```
@@ -19,4 +20,12 @@ RUN NVJPEG2K_VERSION=0.1.0 && \
     apt-get install libnvjpeg2k0 libnvjpeg2k-dev -y && \
     cp /usr/include/nvjpeg2k* /usr/local/cuda/include/ && \
     cp /usr/lib/x86_64-linux-gnu/libnvjpeg2k* /usr/local/cuda/lib64/ && \
-    rm -rf /var/lib/apt/lists/*
+    rm -rf /var/lib/apt/lists/* && \
+    mkdir nvcomp && \
+    cd nvcomp && \
+    wget https://developer.download.nvidia.com/compute/nvcomp/${NVCOMP_VERSION}/local_installers/nvcomp_install_CUDA_11.x.tgz && \
+    tar -xvf nvcomp*.tgz && \
+    cp -rv include/nvcomp* /usr/local/cuda/include/ && \
+    cp -v lib/*.so /usr/local/cuda/lib64/ && \
+    cd .. && \
+    rm -rf nvcomp
```

```suggestion
    rm -rf nvcomp*
```
To remove nvcomp*.tgz as well.

---

I wget `nvcomp*.tgz` into nvcomp dir that is deleted here.